### ISSUE TITLE
Отказаться от опкода LINE_NUM в пользу таблицы с номерами строк и офсетами

### ISSUE THREAD

**Issue by otymko** [otymko/JOneS#140]
Может дать прирост производительности

**Comment by EvilBeaver** [comment]
Все бы ничего, но на этом опкоде работает отладчик и покрытие тестами

**Comment by otymko** [comment]
@EvilBeaver да, поэтому придется "попотеть". С другой стороны, никто не мешает в режиме дебага вернуть эти строки.
